src/core/components/SearchChatHistory.module.css

```diff
@@ -68,6 +64,7 @@
   border-radius: var(--borderRadius-sm);
   padding: 0 var(--spacing-md);
   color: var(--colors-textPrimary);
+  padding-right: calc(var(--spacing-md) * 3);
```

Comment:
We want to style the close button inside the input, but we need enough padding so that on hover, there is no collision with the text.